3-projects/folders.tf

```diff
@@ -14,7 +14,19 @@
  * limitations under the License.
  */
 
-resource "google_folder" "app" {
-  parent       = "organizations/${var.organization_id}"
-  display_name = "app"
+/******************************************
+  Business Unit Folders
+ *****************************************/
+resource "google_folder" "business_unit" {
```

Comment:
nit: to help clearly label what this is would be good to have something like `example_business_unit` for resource name